### PR TITLE
A fix for a spelling mistake

### PR DESCRIPTION
**Pull request by IdanDor**
In the debugger the button which lets you refresh the registers has the word "refrash" instead of "refresh", the fix should be (I think this should work) changing the line:
btnRefrash = new JButton("Refrash");
into this:
btnRefrash = new JButton("Refresh");
this is from the file CpuFrame.java from the gui folder.

### CODE REVIEW

**Comment by rmkn85** [comment]
@kirill578 hi, i want to merge some of your commits, are you still active here?

**Comment by kirill578** [comment]
I'm still here, though I haven't changed anything in here for over a year,
feel free to send pull request

Kirill

**Comment by rmkn85** [comment]
you sent a pull request, i want to go over it together and merge into master

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 corewars8086
 ============
 
-Core Wars for standard 8086 assembly.
\ No newline at end of file
+Core Wars for standard 8086 assembly.
+
+[download latest release](https://github.com/kirill578/corewars8086/releases/tag/Nightly-2014-02-12)
```
Comment (rmkn85) [inline README.md]: No reason to merge this into master, right?

File: corewars8086/nasm/LICENSE [hunk 2]
```diff
@@ -0,0 +1,29 @@
+NASM is now licensed under the 2-clause BSD license, also known as the
```
Comment (rmkn85) [inline corewars8086/nasm/LICENSE]: why distribute the nasm windows exe with the codeguru source code?

**Pull request merged**